engine/classes/ElggGroup.php

```diff
@@ -391,12 +391,16 @@ public function getContentAccessMode() {
 	/**
 	 * Set the content access mode used by group_gatekeeper()
 	 *
-	 * @param string $mode One of CONTENT_ACCESS_MODE_* constants
+	 * @param string $mode One of CONTENT_ACCESS_MODE_* constants. If empty string, mode will not be changed.
 	 * @return void
 	 * @access private
 	 * @since 1.9.0
 	 */
 	public function setContentAccessMode($mode) {
+		if (!$mode && $this->content_access_mode) {
```

Comment:
Note: I chose to do this because this requires checking the raw `->content_access_mode` metadata and I'd prefer not do that in the action.